tajo-jdbc/src/main/java/org/apache/tajo/jdbc/TajoStatement.java

```diff
@@ -319,7 +374,9 @@ public void setCursorName(String name) throws SQLException {
    * Not necessary.
    */
   @Override
-  public void setEscapeProcessing(boolean enable) throws SQLException {}
+  public void setEscapeProcessing(boolean enable) throws SQLException {
```

Comment:
Throwing SQLFeatureNotSupportedException may cause JDBC applications to not work. It would be better to print out a warning message.